src/components/token/TokenBalances.js

```diff
@@ -15,6 +15,9 @@ import { withRouter } from "react-router-dom";
 import ErrorMessageWithIcon from '../error/ErrorMessageWithIcon'
 import TokenAutoCompleteField from './TokenAutoCompleteField';
 import helpers from '../../utils/helpers';
+import { constants as hathorLibConstants } from '@hathor/wallet-lib';
+
+
```

Comment:
[Nitpick] Too much lines before the class

Reply:
Thanks. Removed here https://github.com/HathorNetwork/hathor-explorer/pull/201/commits/ee400af32ff7f520d24a543e1842c4612405ac15